content/100-getting-started/01-quickstart-prisma-postgres.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Quickstart'
+metaTitle: 'Quickstart with TypeScript & Prisma Postgres'
+metaDescription: 'Get started with Prisma ORM in 5 minutes. You will learn how to send queries to a SQLite database in a plain TypeScript script using Prisma Client.'
+search: true
+dbSwitcher: ['sqlite', 'prisma-postgres']
+slugSwitch: /getting-started/quickstart-`
+sidebar_custom_props: { badge: '5 min' }
+---
+
+In this Quickstart guide, you'll learn how to get started from scratch with Prisma ORM and a remote **Prisma Postgres** database in a plain **TypeScript** project. It covers the following workflows:
```

Comment:
@nikolasburk Do you think "remote" is needed here? It somehow implies to me that there's a non-remote PPG as well.

Reply:
good catch, I dropped "remote" here 👍